platform-server-kernel/src/main/java/uno/cod/platform/server/core/security/SecurityService.java

```diff
@@ -79,10 +82,25 @@ public boolean isOrganizationAdmin(User user, UUID organizationId) {
         return false;
     }
 
+    public boolean canAccessChallenge(String user, String canonicalName) {
```

Comment:
We already have `canAccessChallenge(User user, String canonicalName)` what do we need a variant with `String user` for?